### DISCUSSION TITLE
Wrong prefix if dependency has also a resolution.

### DISCUSSION THREAD

**Discussion by jayree** [renovatebot/renovate#28270] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

With the configuration `"config:best-practices"` a PR for a dependency with a resolution looks like `chore(deps): update dependency ...` instead of `fix(deps): update dependency`.

See: https://github.com/jayree/renovate-test/pulls

Is this a bug or did I miss something?

Thanks.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
If a PR has more than one update inside it, the prefix for the "first" update in the PR gets used. In this case it looks like the resolutions one is first.

**Reply by jayree** [reply]
@rarkins there is only one commit inside of the PR: https://github.com/jayree/renovate-test/pull/4/commits I've created another dependency to validate and there is also only one commit with prefix `chore` (https://github.com/jayree/renovate-test/pull/6)

sorry for the confusion. The first Pull Request contains additional updates due to the setup of the example.

**Reply by rarkins** [reply]
There's always one commit even if multiple updates

**Reply by jayree** [reply]
@rarkins this is the correct example: https://github.com/jayree/renovate-test/pull/6 

Is it possible to configure, that a pull request / commit gets the prefix `fix` instead of `chore`? 

Or is this a "bug" because the logic which handles `resolutions` comes first and sets prefix `chore`?

Dependencies which are upgraded and marked with `chore` are not listed in the changelog. And due to the configuration of build pipelines, set to trigger a new release only upon detecting commits starting with `fix` or `feat`, the pipelines are not triggered for this update.

**Reply by rarkins** [reply, marked as answer]
The best solution for you right now is to either:
- Configure all resolutions to use "fix", or
- Configure selective resolutions by package name as "fix"

**Reply by jayree** [reply]
@rarkins nice, that is a good workaround ;)
